Backend/.env

```diff
@@ -1,2 +1,2 @@
-PORT=6969
+PORT=5000
```

Comment:
Add `.env` in `.gitognore`, it should not be available on github.

Reply:
Done.